modules/ROOT/pages/mule-app-dev.adoc

```diff
@@ -0,0 +1,138 @@
+= Mule Application Development
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Mule applications enable you to perform data integrations for your use cases.
+Typical Mule applications read data from internal and external sources,
+process and transform data to the required formats and structures, and
+write that output to the systems and servers where you store and use the
+transformed data.
+
+=== Development Environments
+
+You can develop a Mule application using Anypoint Studio (an Eclipse-based IDE),
+Flow Designer (a Cloud-based application in Design Center, on Anypoint Platform),
+or, if you are an advanced developer, in your own IDE.
+
+For example, in Studio, you build and design a Mule application in a project
+that contains one or more an XML-based files. A Mule project supports all the
+dependencies required for development. For example, the
+xref:7.3@docs-studio::index.adoc#package-explorer[Package Explorer] in Studio
+provides access to the project folders and files that make up a Mule project
+support a design-time environment in which you can also build, run, and test
+your Mule app. Flow Designer supports a Cloud-based version of a Mule project.
+
+//TITLE TBD:
+=== Mule Application Features
+
+It is important to understand concepts related to Mule application development.
+
+* xref:mule-application-about.adoc[Mule Apps]
+
+== Tutorials
+
+These tutorials show a basic Hello World application concepts and a more
+complex integration application that transforms the data it receives over a
+database connection.
+
+* xref:mule-app-dev-helloworld.adoc[Getting Started with Mule Apps: Hello World]
+* xref:mule-app-tutorial.adoc[Mule App Development Tutorial]
+
+== Build and Consume an API
+
+You can build Mule REST or SOAP APIs from an API specification using APIkit,
```

Comment:
As indicated above, I would tell WHY the audience wants/needs an API as part of developing an app, which is the topic of this doc. The rest is fine, or can be moved to that topic that follows the landing page. But I would axe the xrefs, because of the left pane.